common/src/main/webapp/less/style.less

```diff
@@ -1129,6 +1129,11 @@ tfoot {
   margin-left: 0.5em;
 }
 
+.input-replacement-icon {
+  line-height: @inputLineHeight;
+  padding: 0.25em 0;
```

Comment:
I'm looking for the SUI variable that sets the margin in the forms, but haven't found it yet. Still looking...